crates/ar-cli/src/cli.rs

```diff
@@ -33,12 +26,22 @@ pub enum Command {
         /// Example: --seed 3fd54831f488a22b28398de0c567a3b064b937f54f81739ae9bd545967f3abab
         #[arg(long, value_parser = ValueParser::new(parse_32byte_array))]
         seed: [u8; 32],
+    },
+
+    InspectMnemonic {
+        /// 12-word mnemonic phrase
+        #[arg(value_parser)]
```

Comment:
why do we use `value_parser` here?

Reply:
Honest reason: because I was copy-pasting shit, and missed it. Fixed :P